client/my-sites/stats/post-trends/style.scss

```diff
@@ -224,3 +189,13 @@ $mobile-container-margin: 16px;
 		}
 	}
 }
+
+.stats-heat-map__legend {
+	margin-bottom: 48px;
+}
+
+.stats-heat-map__legend-item {
```

Comment:
This is also being applied to other sections where the `StatsHeatMapLegend` is used